x/application/client/cli/tx_stake_application_test.go

```diff
@@ -38,96 +39,114 @@ func TestCLI_StakeApplication(t *testing.T) {
 		fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(net.Config.BondDenom, sdkmath.NewInt(10))).String()),
 	}
 
+	defaultConfig := `
+		service_ids:
+		  - svc1
+		  - svc2
+		  - svc3
+		`
+
 	tests := []struct {
-		desc             string
-		address          string
-		stakeString      string
-		serviceIdsString string
-		err              *sdkerrors.Error
+		desc        string
+		address     string
+		stakeString string
+		config      string
+		err         *sdkerrors.Error
```

Comment:
NIT: s/err/expectedErr

I'm trying to understand what's `input` and what's `output` as I look at this test and it's unclear